### PR TITLE
 adding files

### PR DESCRIPTION
**Pull request by kritika81**

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+<html lang="en">
+<head>
+
+  <title>Bootstrap Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <style>
```
Comment (OnShuffle) [inline app/views/welcome/index.html.erb:12]: move it to application.css

Comment (OnShuffle) [reply app/views/welcome/index.html.erb:12]: <style> tag nahi chahiye wahan

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.1.2'
```
Comment (OnShuffle) [inline Gemfile:10]: rails 4 use krne ko kahan tha meine
chal koi ni.....

**Comment by OnShuffle** [comment]
Comment krna us line mein jahan doubt h

aur sare changes krne ke baad pull request merge kr dena

File: app/views/welcome/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      <span class="sr-only">Next</span>
+    </a>
+</div>
+  
+<div class="container text-center">    
+  <h3>Aspects of happiness</h3><br>
+  <div class="row">
+    <div class="col-sm-4">
+      <img src="https://yt3.ggpht.com/-cOP2X6pxo9M/AAAAAAAAAAI/AAAAAAAAAAA/Dn2UTJzgFkU/s900-c-k-no-mo-rj-c0xffffff/photo.jpg" class="img-responsive" style="width:100%" alt="Image">
+      <p style="font-weight:bold;">Meditation</p>
```
Comment (kritika81) [inline app/views/welcome/index.html.erb:107]: oye..mjhe ye jo meditation likha h..usme click kru to nya page khule...aisa bnana h...steps bta..ror mai btana

Comment (OnShuffle) [reply app/views/welcome/index.html.erb:107]: jo tu chati h use kehete h "link" dena ha meine meditation pe
link ek hi tareeke se diya jata h html pe
use kehete h anchor tag <a></a>
ab w3 schools mein anchor tag pad, uske kuch attributes hote h...

yahan pe <a href="/new_page_link">Meditation</a>

href attribute mein wo dete h jahan link krna h

Comment (OnShuffle) [reply app/views/welcome/index.html.erb:107]: abe ye to link bna diya isne`<a href="/new_page_link">Meditation</a>`

Comment (kritika81) [reply app/views/welcome/index.html.erb:107]: ye mjhe aata h...ror mai bta...ab next kya kru

Comment (kritika81) [reply app/views/welcome/index.html.erb:107]: dusra page bnana h jise link krna h meditation se....ek nya controller bnau?

Comment (OnShuffle) [reply app/views/welcome/index.html.erb:107]: han controller bna

Comment (OnShuffle) [reply app/views/welcome/index.html.erb:107]: ruk

Comment (OnShuffle) [reply app/views/welcome/index.html.erb:107]: pehele is PR ko merge kr de

**Pull request merged**